plugins/modules/iso_extract.py

```diff
@@ -67,6 +67,11 @@
     - The path to the C(7z) executable to use for extracting files from the ISO.
     - If not provided, it will assume the value V(7z).
     type: path
+  password:
+    description:
+    - Password used to decrypt files from the ISO.
+    type: str
+    default: ''
```

Comment:
It's better to not have a default instad of a fake one (empty string). Also, you need to add `version_added`:
```suggestion
    version_added: 10.1.0
```